src/pages/home/report/ReportActionsList.tsx

```diff
@@ -364,12 +364,16 @@ function ReportActionsList({
         (isFromCurrentUser: boolean) => {
             // If a new comment is added and it's from the current user scroll to the bottom otherwise leave the user positioned where
             // they are now in the list.
-            if (!isFromCurrentUser || !hasNewestReportActionRef.current) {
+            if (!isFromCurrentUser) {
+                return;
+            }
+            if (!hasNewestReportActionRef.current) {
+                Navigation.navigate(ROUTES.REPORT_WITH_ID.getRoute(report.reportID));
```

Comment:
We are missing handling the case when the chat is on RHN, which caused [this issue](https://github.com/Expensify/App/issues/50419) with the report on RHN navigating to the center.